api/envoy/config/route/v3/route_components.proto

```diff
@@ -793,6 +820,63 @@ message RouteAction {
 
   reserved "request_mirror_policy";
 
+  // Indicates that some headers received from upstream cluster need to be rewritten. When upstream
+  // returns a header, the header will be overwritten with the given value before being sent back as
+  // response to the client.
+  //
+  // In the below example we rewrite the location header received from the upstream cluster. In the
+  // case of the location header, two things need to match before a value substitution happens for
+  // the location header - Location header's value, and a few other headers present in upstream
+  // response.
+  //
+  // If a header is specified in response_headers_to_rewrite section but is not returned from the
+  // upstream it it simply ignored.
+  //
+  //   .. code-block:: yaml
+  //
+  //     - match:
+  //         prefix: "/itm/"
+  //       route:
```

Comment:
Now that we're using standard matchers, I think this example can be a lot shorter.

Reply:
Done